src/components/__tests__/Dots.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Dots from '../Dots';
+import { mount } from 'enzyme';
+
+const QUANTITY = 2;
+
+describe('Dots Component', () => {
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<Dots />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+  it('checks quantity', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<Dots quantity={QUANTITY} />, div);
```

Comment:
you need to check if wrapper has 2 `dots-container`, otherwise this tests is not validating anything